cheatle/__main__.py

```diff
@@ -1,2 +1,19 @@
+import click
+import re
+
+#validate user input for known
```

Comment:
Avoid unformatted method documentation like this. If you need to write a method comment then let's use the built in doc format that VS code has. Under the method type """ and then enter. It will look like this:

```python
def validate_input(ctx, known, value):
  """_summary_

  Args:
      ctx (_type_): _description_
      known (_type_): _description_
      value (_type_): _description_

  Raises:
      click.BadParameter: _description_

  Returns:
      _type_: _description_
  """
```
 
You need to fill in the placeholders. That being said, I don't think this method needs a comment.